packages/icons/src/svg/32/copy--file.svg

```diff
@@ -1 +1 @@
-<svg id="icon" xmlns="http://www.w3.org/2000/svg" viewBox="0 0 32 32"><defs><style>.cls-1{fill:none;}</style></defs><title>copy--file</title><path d="M29.43,14.7,23.38,8.58A2,2,0,0,0,22,8H9.94A2,2,0,0,0,8,10v18.1A2,2,0,0,0,9.94,30H28.06A2,2,0,0,0,30,28.05v-12A2,2,0,0,0,29.43,14.7ZM22,10l5.92,6H22Zm-12,18V10H20v6a2,2,0,0,0,2,2h6V28Z" transform="translate(0 0)"/><path d="M4,18H2V4A2,2,0,0,1,4,2H18V4H4Z" transform="translate(0 0)"/><rect id="_Transparent_Rectangle_" data-name="&lt;Transparent Rectangle&gt;" class="cls-1" width="32" height="32"/></svg>
\ No newline at end of file
+<svg id="icon" xmlns="http://www.w3.org/2000/svg" width="32" height="32" viewBox="0 0 32 32"><defs><style>.cls-1{fill:none;}</style></defs><title>copy--file</title><path d="M27.4346,14.6987,21.377,8.5776A1.9488,1.9488,0,0,0,19.998,8H12a2.002,2.002,0,0,0-2,2V29a2.002,2.002,0,0,0,2,2H26a2.002,2.002,0,0,0,2-2V16.0771A1.9755,1.9755,0,0,0,27.4346,14.6987ZM20,10.03,25.9235,16H20ZM12,29V10h6v6a2.0023,2.0023,0,0,0,2,2h6.0006l.0009,11Z" transform="translate(0 0)"/><path d="M6,18H4V4A2.002,2.002,0,0,1,6,2H20V4H6Z" transform="translate(0 0)"/><rect id="_Transparent_Rectangle_" data-name="&lt;Transparent Rectangle&gt;" class="cls-1" width="32" height="32"/></svg>
```

Comment:
are `copy--file.svg' and 'copy.svg` intentionally different?

Reply:
yes the change was a request from Mike and I agree with it. The reasoning is that copy--file is more for document use so it should resemble the document icon a bit more while copy is more general and can be its own thing.